test/fixtures/.shout/config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = require('extend')(require('../../../defaults/config.js'), {
+	prefetch: true
+});
```

Comment:
Note that this will modify the cached `defaults/config.js` object (which is why extending is might be a good idea).